package.json

```diff
@@ -1,10 +1,31 @@
 {
-    "private": true,
-    "name": "o-brand",
-    "devDependencies": {
-        "eslint": "^7.9.0",
-        "eslint-config-origami-component": "^2.1.0",
-        "origami-ci-tools": "^2.1.0",
-        "stylelint-config-origami-component": "^1.0.4"
-    }
+  "name": "@financial-times/o-brand",
+  "version": "0.0.0",
+  "description": "Tools to \"brand\" other Origami components.",
+  "keywords": [],
+  "homepage": "https://registry.origami.ft.com/components/o-brand",
+  "bugs": {
+    "url": "https://github.com/Financial-Times/o-brand/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Financial-Times/o-brand.git"
+  },
+  "license": "MIT",
+  "author": "",
+  "type": "module",
+  "devDependencies": {
+    "eslint": "^7.9.0",
+    "eslint-config-origami-component": "^2.1.0",
+    "origami-ci-tools": "3.0.0-beta.6",
+    "remark": "^12.0.1",
+    "remark-lint": "^7.0.1",
+    "remark-preset-lint-origami-component": "^2.0.0-beta.1",
+    "stylelint": "^13.7.0",
+    "stylelint-config-origami-component": "^1.0.4",
+    "typescript": "^4.0.5"
```

Comment:
:eye: wots all this then

Reply:
This was from a prototype I did to see if we could use typescript to build .d.ts files from our JSDoc comments and if those .d.ts files would then be used automatically by code-editors

It worked in vscode, but, I never tried other code-editors

Reply:
I've removed typescript dependency as it is not being used. We can have a discussion in the future about whether we would like to publish our jsdoc as typescript types